.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -14,7 +14,7 @@ for each release artifact. Do not try to sign all of them at once by globbing. I
 
  - [ ] Create a draft release on Github and upload all the release artifacts and their signatures. Copy and paste the release notes from NEWS here as well.
  - [ ] Publish the release
- - [ ] Vendor the new Ignition version in [mantle](https://github.com/coreos/mantle)
+ - [ ] Vendor the new Ignition version in [mantle](https://github.com/coreos/mantle) (backporting to the `cl` branch if a new 0.x.y release is being cut)
```

Comment:
Optional: alternatively, move and duplicate this instruction into the separate sections with the `0.x.y` section explicitly saying the `cl` branch and the `2.x.y+` section the `master` branch?

Reply:
The `master` branch has to be updated in both cases.

Reply:
Ahh right, I forgot it was vendoring both.